packages/driver/src/cy/assertions.ts

```diff
@@ -275,27 +192,26 @@ export const create = (Cypress: ICypress, cy: $Cy) => {
 
   return {
     finishAssertions,
+    assert,
 
     // TODO: define the specific type of options
     verifyUpcomingAssertions (subject, options: Record<string, any> = {}, callbacks: VerifyUpcomingAssertionsCallbacks = {}) {
```

Comment:
`verifyUpcomingAssertions` is no longer an accurate name, but given that a variety of plugins and custom commands rely on it, I've opted not to rename it something more appropriate (like `verifyImplicitAssertions` or somesuch).

It may not be documented as part of our public API, but I'd consider it public-facing and am therefore hesitant to rename it without an extremely compelling reason.

Reply:
Yeah if anything we could rename to verifyImplictAssertions, then update verifyUpcomingAssertions to pass through to the new function and log a warning / use typescript to deprecate the func to remove at some point down the line 🤷🏻